com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/GlobalLightingQualitySettings.cs

```diff
@@ -228,7 +241,12 @@ internal GlobalLightingQualitySettings()
         public int[] RTRDenoiserRadius = new int[s_QualitySettingCount];
 
         // TODO: Volumetric fog quality
-
+        /// <summary>Controls which control mode should be used to define the volumetric fog parameters.</summary>
+        public FogControl[] Fog_ControlMode = new FogControl[s_QualitySettingCount];
+        /// <summary>Controls the budget of the volumetric fog effect.</summary>
+        public float[] Fog_Budget = new float[s_QualitySettingCount];
+        /// <summary>Controls how the budget is shared between screen resolution and depth.</summary>
+        public float[] Fog_DepthRatio = new float[s_QualitySettingCount];
```

Comment:
Hey, so it seems we currently only added quality presets for the **Balance** mode. (**Fog Budget, DepthRatio**), however the **Manual** mode still has none. Is it intended? I think we will need to add them, to comply with the Quality UI pattern, otherwise it will be unclear to users if they change quality preset and nothing happens.

Reply:
Yes it is intended, the manual mode is not used for the preset as it is considered "Advanced" so having default values for it doesnt make sense to me.